aztk/client.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, secrets_config: models.SecretsConfiguration):
         self.blob_client = azure_api.make_blob_client(secrets_config)
 
 
+
+    def get_cluster_data(self, cluster_id: str) -> cluster_data.ClusterData:
```

Comment:
This is meant to be for internal use only, right? So maybe make the function `_get_cluster_data` instead.